README.md

```diff
@@ -221,7 +221,7 @@ The both defaults badge (noob) already was added to all users that we already ha
 Inside your application if you want to give 100 points to some user, inside your function you have to use the following method:
 
 ```ruby
-kind =  Kind.where(:name => "teacher")
```

Comment:
I'm not sure if this example is correct, because where would return a collection, and not the ActiveRecord object itself, therefore the `kind.id` bellow would fail.